layout-management-webapps/src/main/webapp/WEB-INF/gatein-resources.xml

```diff
@@ -90,6 +90,35 @@
     </module>
   </portlet>
 
+  <module>
+    <name>manageAccessPermission</name>
```

Comment:
```suggestion
    <name>managePermission</name>
```

Reply:
it is already completely changed in the other PR based on this! 
I suggest keeping this request for the other PR